hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationService.java

```diff
@@ -42,6 +42,8 @@
 
     <T> T toObject(Object data);
 
+    <T> T bytesToObject(byte[] bytes, int dataOffset);
```

Comment:
Can we add also int length here ?
or separate method
+    <T> T bytesToObject(byte[] bytes, int dataOffset, int length);
?